app/models/user.rb

```diff
@@ -16,9 +18,9 @@ class User < ActiveRecord::Base
   attr_accessible *(ACCESSIBLE_ATTRIBUTES + [:admin]), :as => :admin
 
   validates_presence_of :username
-  validates_uniqueness_of :username
+  validates :username, uniqueness: { case_sensitive: false }
```

Comment:
:facepalm: Right, the change is for postgresql which does case-sensitive comparisons, with `case_sensitive: false` the query becomes `SELECT  1 AS one FROM "users" WHERE LOWER("users"."username") = LOWER('Admin') LIMIT 1`.